src/test/java/org/broadinstitute/hellbender/tools/copynumber/PostprocessGermlineCNVCallsIntegrationTest.java

```diff
@@ -1,73 +1,209 @@
+            IntegrationTestSpec.assertEqualTextFiles(actualSegmentsOutputVCF, expectedSegmentsOutputVCF);
+        }
+    }
+
+    @Test(dataProvider = "differentValidInput")
+    public void testDifferentValidInputIntervalsVCFOnly(final List<String> callShards,
+                                                        final List<String> modelShards) throws IOException {
+        for (int sampleIndex = 0; sampleIndex < NUM_TEST_SAMPLES; sampleIndex++) {
+            final File actualIntervalsOutputVCF = createTempFile("intervals-output-vcf-" + sampleIndex, ".vcf");
+            final File expectedIntervalsOutputVCF = INTERVALS_VCF_CORRECT_OUTPUTS.get(sampleIndex);
+            runToolForSingleSample(callShards, modelShards, sampleIndex,
+                    actualIntervalsOutputVCF, null,
+                    ALLOSOMAL_CONTIGS, AUTOSOMAL_REF_COPY_NUMBER, false);
+            IntegrationTestSpec.assertEqualTextFiles(actualIntervalsOutputVCF, expectedIntervalsOutputVCF);
+        }
+    }
+
+    @Test(dataProvider = "differentInvalidInput")
+    public void testDifferentInvalidInput(final List<String> callShards,
+                                          final List<String> modelShards) throws IOException {
+        /* dry run -- just input data validation */
+        for (int sampleIndex = 0; sampleIndex < NUM_TEST_SAMPLES; sampleIndex++) {
+            try {
+                runToolForSingleSample(callShards, modelShards, sampleIndex,
+                        createTempFile("intervals-output-vcf", ".vcf"),
+                        createTempFile("segments-output-vcf", ".vcf"),
+                        ALLOSOMAL_CONTIGS, AUTOSOMAL_REF_COPY_NUMBER, true);
+            } catch (final IllegalArgumentException ex) {
+                continue;
+            }
+            throw new AssertionError("The test should have failed on invalid input.");
+        }
+    }
+
+    @Test(expectedExceptions = IllegalArgumentException.class)
+    public void testBadAutosomalContigs() {
+        runToolForSingleSample(CALL_SHARDS, MODEL_SHARDS, 0,
+                createTempFile("intervals-output-vcf", ".vcf"),
+                createTempFile("segments-output-vcf", ".vcf"),
+                Collections.singletonList("Z"), /* unknown contig */
+                AUTOSOMAL_REF_COPY_NUMBER, true);
+    }
+
+    @Test(dataProvider = "badRefAutosomalCopyNumber")
+    public void testOutOfRangeRefState(final int badRefAutosomalCopyNumber) {
+        try {
+            runToolForSingleSample(CALL_SHARDS, MODEL_SHARDS, 0,
+                    createTempFile("intervals-output-vcf", ".vcf"),
+                    createTempFile("segments-output-vcf", ".vcf"),
+                    ALLOSOMAL_CONTIGS,
+                    badRefAutosomalCopyNumber, /* out of range */
+                    true);
+        } catch (final CommandLineException.OutOfRangeArgumentValue | IllegalArgumentException ex) {
+            return;
```

Comment:
Same, use `expectedExceptions` and throw accordingly.

Reply:
One can not catch multiple exceptions using `expectedExceptions`. I could break the test into multiple tests. I removed the test anyway (I don't check for out of range ref state anymore -- it was an arbitrary test).